src/main/java/io/github/mountainrange/mule/gameplay/Grid.java

```diff
@@ -207,4 +209,35 @@ public void removePlayer() {
 	public Point2D getPlayerPosition() {
 		return playerPosition;
 	}
+
+	@Override
```

Comment:
Just a heads up, this iterator will only go over the tiles, not the players. We decided to store the player independently for the tiles so they could eventually move around freely. Good idea though!